packages/react-core/src/components/Modal/ModalBoxTitle.tsx

```diff
@@ -1,12 +1,27 @@
 import * as React from 'react';
 import modalStyles from '@patternfly/react-styles/css/components/ModalBox/modal-box';
 import { css } from '@patternfly/react-styles';
-
+import accessibleStyles from '@patternfly/react-styles/css/utilities/Accessibility/accessibility';
+import { capitalize } from '../../helpers';
 import { Tooltip } from '../Tooltip';
+import CheckCircleIcon from '@patternfly/react-icons/dist/js/icons/check-circle-icon';
+import ExclamationCircleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-circle-icon';
+import ExclamationTriangleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-triangle-icon';
+import InfoCircleIcon from '@patternfly/react-icons/dist/js/icons/info-circle-icon';
+import BellIcon from '@patternfly/react-icons/dist/js/icons/bell-icon';
+
+export const isModalAlertVariant = (icon: any): icon is string =>
+  ['success', 'danger', 'warning', 'info', 'defaut'].includes(icon as string);
```

Comment:
Can you rename this to `isVariantIcon`? Very clever usage of `icon is string`, I've never seen that before!